File: client/src/app/_models/orders/orderForwardCategoryOfficial.ts
export interface IOrderForwardCategoryOfficial
{
	checked: boolean;
	id: number;
	orderForwardCategoryId: number;
	customerOfficialId: number;
	officialName: string;
	agentName: string;
	dateForwarded: Date;
	//dateOnlyForwarded: Date;
	emailIdForwardedTo: string;
	phoneNoForwardedTo: string;
	whatsAppNoForwardedTo: String;
	username: string;
}

export class OrderForwardCategoryOfficial implements IOrderForwardCategoryOfficial
{
	checked = false;
	id= 0;
	orderForwardCategoryId= 0;
	agentName= '';
	customerOfficialId= 0;
	officialName = '';
	dateForwarded= new Date('1900-01-01');
	//dateOnlyForwarded= new Date('1900-01-01');
	emailIdForwardedTo= '';
	phoneNoForwardedTo= '';
	whatsAppNoForwardedTo= '';
	username= '';

	orderItemId= 0;
}

File: api/Entities/Admin/Order/OrderForwardCategoryOfficial.cs
namespace api.Entities.Admin.Order
{
    public class OrderForwardCategoryOfficial: BaseEntity
    {
        public int OrderForwardCategoryId { get; set; }
        public int CustomerOfficialId { get; set; }
        public string OfficialName { get; set; }
        public int CustomerId { get; set; }
        public string AgentName {get; set;}
        public DateTime DateForwarded {get; set;}
        public string EmailIdForwardedTo { get; set; }
        public string PhoneNoForwardedTo { get; set; }
        public string WhatsAppNoForwardedTo { get; set; }
        public string Username {get; set;}
    }
}